modules/deployment/src/test/java/org/mule/runtime/module/deployment/internal/DeploymentServiceTestCase.java

```diff
@@ -2812,7 +2815,8 @@ private void assertMuleContextInitialized(final DeploymentListener listener, fin
 
       @Override
       public boolean test() {
-        verify(listener, times(1)).onMuleContextInitialised(eq(appName), any(MuleContext.class));
+        verify(listener, times(1)).onMuleContextInitialised(eq(appName),
+                                                            any(MuleContext.class));
```

Comment:
There are many lines like this one that were changed with not reason, maybe you formatter config is different